src/meta/processors/BaseProcessor.h

```diff
@@ -247,14 +249,16 @@ class BaseProcessor {
 
     void doSyncMultiRemoveAndUpdate(std::vector<std::string> keys);
 
+    bool saveRebuildStatus(std::string statusKey, std::string&& statusValue);
+
     /**
      * check if the edge or tag contains indexes when alter edge or tag.
      **/
     cpp2::ErrorCode indexCheck(const std::vector<nebula::cpp2::IndexItem>& items,
                                const std::vector<cpp2::AlterSchemaItem>& alterItems);
 
     StatusOr<std::vector<nebula::cpp2::IndexItem>>
-    getIndexes(GraphSpaceID spaceId, int32_t edgeOrTag, bool isEdge);
+    getIndexes(GraphSpaceID spaceId, int32_t schemaId);
```

Comment:
what's schemaId?

Reply:
both tag and edge are schema so them called by a joint name `schema ID`

Reply:
Maybe tagOrEdge is a better name.